great-idea-website/css/index.css

```diff
@@ -63,4 +63,70 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+.logo {
+    display: inline-block;
+    margin-left: 100px;
+}
+.container {
+    width: 1000px;
+    margin: 0 auto;
+}
+nav{
+    display: inline;
```

Comment:
To center the navigation bar with the "Great Idea" logo, I've seen `vertical-align` used on the `nav` in conjunction with adding `margin-top` to the image to make them more centered. Of course moving forward, you should use Flexbox to achieve the same thing much more easily.